ui/raidboss/data/04-sb/raid/o7s.js

```diff
@@ -158,7 +159,7 @@
           en: 'Blue Marker on ' + data.ShortName(matches.target),
           de: 'Aura-Kanone auf ' + data.ShortName(matches.target),
           fr: 'Marque Bleue sur ' + data.ShortName(matches.target),
-          ko: '파란징 → ' + data.ShortName(matches.target),
+          ko: '"' + data.ShortName(target) + '" 파란징',
```

Comment:
I think this is incorrect.  `target` should be `matches.target`.  This happens in multiple places.

Reply:
i missed it! thanks